### DISCUSSION TITLE
Tried to create pull request for comments

### DISCUSSION THREAD

**Discussion by doujouDC** [datacarpentry/r-socialsci#435] [Q&A]
Hey Maintainer,

Love the lesson. The link to the json "The json episode uses SAFI.json. The downlink link is https://raw.githubusercontent.com/datacarpentry/r-socialsci/main/data/SAFI.json." does not work, or at least it did not for me. 

Also, I was on the "live" lesson page but when I went to submit a change request I got a "splash screen" telling me "You came to this page from a Frozen snapshot of the lesson" This is the page that flagged the error: https://carpentries.github.io/workbench/contributor/beta.html?id=https://github.com/datacarpentry/r-socialsci/edit/main/setup.md

Also, I was unsuccessful on creating a fork so I could submit a change request. I guess I could not create the pull request because I do not have the correct path to the json file...thus I did not make any changes.

Thanks

Doug

**Comment by zkamvar** [comment]
Hi Doug, 

Thank you for opening this discussion. Because this is related to the beta phase, this is my realm. The reason why the link does not work is because I had missed it when converting the lesson to use The Carpentries Workbench (see #411 and https://carpentries.github.io/workbench/beta-phase.html#beta for an explanation of what goes on in the beta stage). The new link is: https://datacarpentry.org/r-socialsci/data/SAFI.json

It looks like you were successful in creating a fork: https://github.com/doujouDC/r-socialsci, so feel free to fix that link in `instructors/setup.md` so that it is correct on https://preview.carpentries.org/r-socialsci and I will fix the link on the frozen page.

**Reply by zkamvar** [reply]
Additionally, the _reason_ why the link is broken is because the underlying folder structure has changed as shown here: https://carpentries.github.io/workbench/transition-guide.html#folder-structure

**Reply by doujouDC** [reply]
Hi and thanks for taking the time to reply. I am a bit confused by this, so perhaps you might be able to clariy. This link that did not work was on this page: https://datacarpentry.org/r-socialsci/ which does not appear to be in beta. Am I missing something? Also, if I understand I can go ahead and replace this linke with: "https://datacarpentry.org/r-socialsci/data/SAFI.json". I just want to make sure I am not missing something

Doug

**Reply by zkamvar** [reply]
> Also, if I understand I can go ahead and replace this linke with: "https://datacarpentry.org/r-socialsci/data/SAFI.json".

Yes, this is correct. 

> This link that did not work was on this page: https://datacarpentry.org/r-socialsci/ which does not appear to be in beta.

I'd be more than happy to explain. If you prefer a visual explanation, I have a video here: https://www.youtube.com/watch?v=jnvEz-ojlUc

It is part of the beta phase (as indicated in the top banner). We wanted to make sure that people could compare and contrast the lesson structure, [we host the two versions of the lesson in parallel](https://carpentries.org/blog/2023/01/dovetail-14/#entering-the-second-stage-of-the-beta-phase). The old jekyll version of the lesson (now frozen) is still hosted at the default site and the new Workbench version is hosted at https://preview.carpentries.org/r-socialsci/

The tradeoff of hosting the different versions is that it's not possible to edit both at the same time, so we froze the Jekyll version while the community has the opportunity to contribute to and edit the Workbench version (which will become default in April). 

If you clicked on the "edit this page" button and got a 404, that was a bug that I have just fixed in the past couple of hours. It should work now.

**Reply by zkamvar** [reply]
Note that the splash page you encountered will take you to: https://github.com/datacarpentry/r-socialsci/edit/main/learners/setup.md, which is the correct place to edit that link.
